supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorBackendFactory.scala

```diff
@@ -1,21 +1,27 @@
 package cromwell.backend.impl.htcondor
 
-
 import akka.actor.Props
 import cromwell.backend._
+import cromwell.backend.impl.htcondor.caching.CacheActorFactory
 import cromwell.backend.io.{JobPaths, SharedFsExpressionFunctions}
 import cromwell.core.CallContext
 import wdl4s.Call
 import wdl4s.expression.WdlStandardLibraryFunctions
 
 case class HtCondorBackendFactory(configurationDescriptor: BackendConfigurationDescriptor) extends BackendLifecycleActorFactory {
+  val provider = configurationDescriptor.backendConfig.getString("cache.provider")
+  val cacheFactory = Class.forName(provider)
```

Comment:
This should only be evaluated if caching is enabled, so that caches and databases can be decoupled during testing.